Create project with Vue + StandardJS rules, module build failed

run `yarn create @neutrinojs/project vueapp2` with these answers

- A web or Node.js application
- Vue
- Karma
- StandardJS rules

The module build is failed when starting the app.
```
C:\workspace\web\neutrino-test\> cd vueapp2
C:\workspace\web\neutrino-test\vueapp2> yarn start
yarn run v1.1.0
$ neutrino start
√ Development server running on: http://localhost:5000
√ Build completed

ERROR in ./src/index.js
Module build failed: Error: Failed to load plugin react: Cannot find module 'eslint-plugin-react'
Referenced from:
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.resolve (internal/module.js:18:19)
    at Plugins.load (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint\lib\config\plugins.js:106:29)
    at Array.forEach (<anonymous>)
    at Plugins.loadAll (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint\lib\config\plugins.js:146:21)
    at loadFromDisk (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint\lib\config\config-file.js:492:35)
    at load (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint\lib\config\config-file.js:550:20)
    at configExtends.reduceRight (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint\lib\config\config-file.js:421:36)
    at Array.reduceRight (<anonymous>)
    at applyExtends (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint\lib\config\config-file.js:403:28)
    at Object.loadObject (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint\lib\config\config-file.js:529:35)
    at new Config (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint\lib\config.js:72:46)
    at new CLIEngine (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint\lib\cli-engine.js:420:23)
    at Object.module.exports (C:\workspace\web\neutrino-test\vueapp2\node_modules\eslint-loader\index.js:176:27)
 @ multi (webpack)-dev-server/client?http://localhost:5000 (webpack)/hot/dev-server.js ./src/index
```

It will be passed if I comment the `@neutrinojs/standardjs` entry in the `.neutrinorc.js`.

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     "codecov": "^3.0.0",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-prettier": "^2.3.1",
-    "eslint-plugin-react": "^7.4.0",
+    "eslint-plugin-react": "^7.5.1",
     "gh-pages": "^1.1.0",
     "gitbook-cli": "^2.3.2",
     "gitbook-plugin-anchorjs": "^1.1.1",

--- a/packages/preact/package.json
+++ b/packages/preact/package.json
@@ -26,7 +26,8 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-es2015-classes": "^6.24.1",
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "deepmerge": "^1.5.2"
+    "deepmerge": "^1.5.2",
+    "eslint-plugin-react": "^7.5.1"
   },
   "peerDependencies": {
     "neutrino": "^8.0.0"

--- a/packages/vue/package.json
+++ b/packages/vue/package.json
@@ -20,6 +20,7 @@
     "@neutrinojs/web": "^8.0.11",
     "babel-preset-vue": "^2.0.0",
     "deepmerge": "^1.5.2",
+    "eslint-plugin-react": "^7.5.1",
     "eslint-plugin-vue": "^4.0.0-beta.2",
     "stylelint-processor-html": "^1.0.0",
     "vue-loader": "^13.5.0",

